src/test/java/de/berlin/lostberlin/controller/OrderControllerMVCTest.java

```diff
@@ -0,0 +1,218 @@
+package de.berlin.lostberlin.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.berlin.lostberlin.exception.ResourceNotFoundException;
+import de.berlin.lostberlin.model.Order;
+import de.berlin.lostberlin.repository.OrderRepository;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.context.web.WebAppConfiguration;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.util.List;
+
+import static java.util.Collections.singletonList;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultHandlers.print;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+/*
+In order to run MVC tests I had to comment out the annotation @EnableJpaAuditing in the main class.
+Unfortunately I didn't manage to fix it in a more appropriate way. Advises and hints will be highly appreciated!
+ */
+@RunWith(SpringRunner.class)
+@WebAppConfiguration
+@WebMvcTest(OrderController.class)
+public class OrderControllerMVCTest {
+
+    @Autowired
+    private MockMvc mvc;
+
+//    @Autowired
+//    private ObjectMapper mapper;
+
+    @MockBean
+    OrderRepository orderRepo;
+
+    @Test
+    public void test_get_all_orders_success() throws Exception {
+        Order order = mockOrder();
+        List<Order> allOrders = singletonList(order);
+        given(this.orderRepo.findAll()).willReturn(allOrders);
+
```

Comment:
We test `OrderController`, for this we should mock its JPA dependency - `OrderRepository`. Therefore we configure  mock for OrderRepository, but calling OrderController to see whether the logic in it is correct.